Neos.ContentGraph.DoctrineDbalAdapter/Tests/Behavior/Features/Bootstrap/FeatureContext.php

```diff
@@ -57,6 +57,7 @@ public function resetContentRepositoryComponents(BeforeScenarioScope $scope): vo
     }
 
     protected function getContentRepositoryService(
+        ContentRepositoryId $contentRepositoryId,
```

Comment:
Just wondering: why do we have to change the signature? Can't we just refer to `currentContentRepository`?